recipes/_configure_server.rb

```diff
@@ -132,6 +132,38 @@
   }
 end if license_uri == 'self'
 
+license_pools = CernerSplunk::DataBag.load(node['splunk']['config']['license-pool'])
+
+if node['splunk']['node_type'] == :license_server && !license_pools.nil?
+
+  # Find the total available license quota
+  total_available_license_quota = 0
+  existing_license = Dir.glob("#{node['splunk']['home']}/etc/licenses/enterprise/*.lic*")
+  existing_license.each do |file|
+    license_file = Nokogiri::XML(File.open(file))
+    max_quota = license_file.at_xpath('/license/payload/expiration_time/text()').to_s.to_i > Time.now.to_i ? license_file.at_xpath('/license/payload/quota/text()').to_s.to_i : 0
```

Comment:
Fair enough. Their usage documentation for single result searching is misleading then as it indicates that strings are returned.